software/winrm/src/main/java/org/apache/brooklyn/util/core/internal/winrm/WinRmTool.java

```diff
@@ -86,6 +86,8 @@
             "Size of file chunks (in bytes) to be used when copying a file to the remote server", 
             1024);
 
+    ConfigKey<String> ADDITIONAL_CONNECTION_METADATA = newStringConfigKey("additional.connection.metadata", "Can be used by a location customizer during provisioning to provide additional information, which in turn can be consumed by a custom ssh tool");
```

Comment:
Perhaps word as something like "Can be used to pass additional custom data to the `WinrmTool`, which is especially useful if writing a bespoke tool implementation."

Your description is accurate for your use-case, but really it's not specific to the customizer - it could be passed in as normal location config or entity `provisioning.properties`.